src/parser/warrior/arms/modules/core/Overpower.js

```diff
@@ -60,6 +71,28 @@ class OverpowerAnalyzer extends Analyzer {
         .recommended(`${formatPercentage(recommended)}% is recommended.`);
     });
   }
+
+  statistic() {
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.OVERPOWER.id} />}
+        label="Overpower Buffs Wasted"
+        position={STATISTIC_ORDER.OPTIONAL(6)}
+        value={(
+          <>
+              {this.wastedProc} <small>wasted buffs</small><br />
+              {this.overpowerCasts} <small>total casts</small>
+          </>
+        )}
+        tooltip={(
+          <>
+            If Overpower is casted with two buffs applied 
```

Comment:
as a non-warrior it is not obvious to me what buffs you're talking about. would this be obvious to a new arms player?

this is the only issue i see with the PR

Reply:
I re-wrote the tooltip to hopefully be more clear.